Project.toml

```diff
@@ -13,19 +14,19 @@ Tokenize = "0796e94c-ce3b-5d07-9a54-7f471281c624"
 URIParser = "30578b45-9adc-5946-b283-645ec420af67"
 
 [compat]
-CSTParser = "1"
-DocumentFormat = "1.0.1"
+CSTParser = "1.1"
+DocumentFormat = "1.1.1"
 JSON = "0.20, 0.21"
-StaticLint = "1.0.2"
-Tokenize = "0.5.6"
+StaticLint = "1.1"
+Tokenize = "0.5.7"
```

Comment:
I'll update them before I tag a new release, but I'd say we merge as is for now and I can fix on `master` before tagging.